web/public/locales/sv.json

```
@@ -950,15 +950,15 @@
     },
     "IT-CNO": {
       "countryName": "Italien",
-      "zoneName": "Centrala norra"
+      "zoneName": "Centrala norra Italien"
```

```suggestion
      "zoneName": "Centrala Italien norr"
```
or
```suggestion
      "zoneName": "Norra centrala Italien"
```
Same here as the other suggestion.